spec/index.html

```diff
@@ -2383,12 +2381,127 @@ <h4>YamlLdErrorCode</h4>
           </section>
         </section>
 
+        <section>
+          <h2>Implementations</h2>
+
+          TODO
+        </section>
+
       </section>
 
       <section>
         <h2>Convert Extended YAML-LD to Basic YAML-LD and back</h2>
 
-        TODO
+        <p>
+          This approach is simpler than the Extended Internal Representation because it
+          does not require any changes to the internal structures of existing JSON-LD
+          libraries.
+        </p>
+
+        <p>
+          Instead, we implement two API functions:
+        </p>
+
+        <dl data-sort>
```

Comment:
To prevent very complex processing model, it's important that these functions work on the expanded representation, not compacted. Otherwise, keeping track of term definitions and scoped contexts is quite difficult.

Reply:
Also, we'll need to clarify that this is not really a round-trip, as it can't know the specifics of the original syntax usage, rather it "optimizes"  a basic YAML-LD document using extended features.